logging.go

```diff
@@ -209,14 +216,21 @@ func (logging *Logging) Logger(mod Module) *zap.Logger {
 					cores = append(cores, l.core)
 					continue
 				}
+				if len(options) == 0 {
+					newOptions, err := l.buildOptions()
+					if err != nil {
+						Log().Error("building options for logger", zap.String("module", modID), zap.Error(err))
+					}
+					options = newOptions
```

Comment:
This is a non-fatal error log because this call chain doesn't support bubbling up an error. Probably good enough, it's very rare (only if user uses an invalid level arg for `with_stacktrace`).